mne/io/reference.py

```diff
@@ -186,20 +186,31 @@ def add_reference_channels(inst, ref_channels, copy=True):
                         % type(inst))
     nchan = len(inst.info['ch_names'])
 
-    # "zeroth" EEG electrode dig points is reference
-    ref_dig_loc = [dl for dl in inst.info['dig'] if (
-                   dl['kind'] == FIFF.FIFFV_POINT_EEG and
-                   dl['ident'] == 0)]
-    if len(ref_channels) > 1 or len(ref_dig_loc) != len(ref_channels):
+    # only do this if we actually have digitisation points
+    if 'dig' in inst.info and inst.info['dig']:
```

Comment:
usually we are more explicit as `and inst.info['dig'] is not None` or `and len(inst.info['dig']) > 0` depending on which is appropriate (makes the check clearer)